### PR TITLE
Fix small UI inconsistencies in share prescription screen

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Comment by msasikanth** [comment]
Fixing some more small inconsistencies, marking it as WIP

**Comment by vinaysshenoy** [comment]
Assigning this to @Honey14 since I am on leave and the PR is a little important to merge in.

File: app/src/main/res/layout/screen_teleconsult_share_prescription.xml [hunk 1]
```diff
@@ -112,58 +109,63 @@
...
           app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
           app:layout_constraintBottom_toTopOf="@+id/sharePrescriptionMedicinesDivider"
           app:layout_constraintTop_toBottomOf="@+id/medicinesTextView" />
 
         <View
           android:id="@+id/sharePrescriptionMedicinesDivider"
-          style="@style/Clinic.V2.ListSeparator"
-          android:layout_gravity="bottom"
+          android:layout_width="match_parent"
+          android:layout_height="@dimen/spacing_1"
```
Comment (Honey14) [inline app/src/main/res/layout/screen_teleconsult_share_prescription.xml:120]: We are using `ListSeparator` in some places and height spacing as 1dp in some places? Are we going to remove `ListSeparator` later?

**Review by Honey14 — APPROVED** [review]
Looks good. Left one comment

File: app/src/main/res/layout/screen_teleconsult_share_prescription.xml (hunk 1 above)
Comment (msasikanth) [reply app/src/main/res/layout/screen_teleconsult_share_prescription.xml:120]: `ListSeparator` was a older implementation, that style has margin set to it by default. During material themeing changes I will try to create custom `DividerView` that would simplify this.

**Pull request merged**